pkgs/tools/misc/svtplay-dl/default.nix

```diff
@@ -43,9 +43,10 @@ in stdenv.mkDerivation rec {
 
   meta = with stdenv.lib; {
     homepage = "https://github.com/spaam/svtplay-dl";
-    description = "Command-line tool to download videos from svtplay.se and other sites";
+    description =
+      "Command-line tool to download videos from svtplay.se and other sites";
```

Comment:
```suggestion
    description = "Command-line tool to download videos from svtplay.se and other sites";
```

Reply:
FYI, this was actually introduced by `nixfmt`!